doc/source/user_reference/output_directory.rst

```diff
@@ -0,0 +1,132 @@
+.. _output_directory_structure:
+
+Output Directory Structure
+==========================
+
+This is an over overview of WA output directory structure.
```

Comment:
Do you mean just "overview"? Or maybe "high-level overview"?

Reply:
Erm, what's the difference in this context?